### PR TITLE
Feature/geospatial tweaks

### PR DESCRIPTION
**Pull request by katenorquay**
https://trello.com/c/pinrc5Cp/1578-abletech-website-geospatial-showcase-page

### CODE REVIEW

**Comment by katenorquay** [comment]
desktop_geospatial
mobile_geospatial

File: source/work/services/mapping.html.erb [hunk 1]
```diff
@@ -8,20 +8,22 @@ page_classes: simple_page services_section service_page_mapping work
...
-      How can your statistics reveal the big picture for your business?
+      Abletech specialises in building user-friendly systems that convey location data visually.<br>
+      Whether you want to know where things are, map your business locations, or visualise vast quantities of data to see patterns, we can help.
     </p>
   </div>
 </section>
-<section class="section section_column bordered_section">
-  <p>Abletech specialises in building user-friendly systems that convey location data visually. Whether you want to know where things are, map your business locations, or visualise vast quantities of data to see patterns, we can help.</p>
-  <p><a href="mailto:hello@abletech.nz" class="button">Get in touch!</a></p>
+<section class="section section_column section_column_banner section_neutral">
```
Comment (katenorquay) [inline source/work/services/mapping.html.erb]: @AlexB52 This section has the updated top call to action text, 'Are you looking for a geospatial solution' and the "paragraph paragraph_intro" class makes the text bolder

File: source/work/services/mapping.html.erb [hunk 2]
```diff
@@ -134,7 +136,7 @@ page_classes: simple_page services_section service_page_mapping work
 </section>
 <section class="section section_column section_column_banner section_alternate">
   <div class="banner_container">
-    <h1 class="paragraph paragraph_intro">We’d love to talk with you about your online mapping or geodata visualisation ideas.</h1>
+    <h1 class="paragraph paragraph_intro">We’d love to talk with you about your next project, and how we can make it a success</h1>
```
Comment (katenorquay) [inline source/work/services/mapping.html.erb]: this change is updating the bottom call to action. Perhaps calling it a 'call to action' in the trello card made it confusing?

File: source/work/services/mapping.html.erb (hunk 1 above)
Comment (AlexB52) [reply source/work/services/mapping.html.erb]: Right I thought the top call to action was the button not the text above.

Perfect then 👍

**Pull request merged**